demos/demos_databases_apis/azure_data_explorer/graphistry_ADX_kusto_demo.ipynb

```diff
@@ -0,0 +1,469 @@
+    "\n",
+    "The kusto_graph function accepts two parameters. \n",
+    "The name of the graph, and the name of your snapshot **(snap_name=\"name\")**. If you don't provide a snapshot it will grab the latest snapshot.\n",
+    "\n",
+    "The function returns a Graphistry plottable object.\n",
+    "\n",
+    "You can inspect the nodes and edges, add customizations or .plot() it as is."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b7b51e72-cc58-4ef8-8189-4353fc2f87cc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "g = graphistry.kusto_graph(graph_name, snap_name=snapshot_name)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "46d71bbb-92ec-41cb-86e7-0296cc02dc8a",
+   "metadata": {},
+   "source": [
+    "## Plotting your object"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e4aa923b-bdb6-4104-9a0f-348487fd9041",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "g.plot()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f0fb5c7e-5356-4448-ac4b-2887f00529d2",
+   "metadata": {},
+   "source": [
+    "### Changing colors, icons and more"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a23bda5b-1f39-46f0-a634-73071c3c512d",
+   "metadata": {},
+   "source": [
+    "#### Encode point color\n",
+    "\n",
+    "\n",
+    "Our data consists of two datasets where one contains verified red team activity. In the dataset these are tagged with the value 1 in the column **RED**.\n",
+    "\n",
+    "Let's make our red nodes pop out in our visualization."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0d15a4b9-431f-4b74-bc3c-dbfc4bf3d637",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "g2 = g.encode_point_color(\n",
+    "    \"RED\",\n",
+    "    categorical_mapping={\n",
+    "        1: \"red\"\n",
+    "    },\n",
+    "    default_mapping='blue'\n",
+    ")\n",
+    "\n",
+    "g2.plot()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f3c5f1bd-ce65-4dce-b6f9-9aae48527727",
+   "metadata": {},
+   "source": [
+    "#### Encode icons\n",
+    "\n",
+    "\n",
+    "As our data is split into two different type of nodes **\"Computer\"** and **\"Domain\"**.\n",
+    "\n",
+    "Give the different node types it's own Icon."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "24201fbf-53a3-45dc-b235-cc45abc7d923",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "g3 = g2.encode_point_icon(\n",
+    "    'nodeType',\n",
+    "    shape=\"circle\",\n",
+    "    categorical_mapping={\n",
+    "        \"Computer\": \"laptop\", \n",
+    "        \"Domain\": \"server\"\n",
+    "    },\n",
+    "    default_mapping=\"question\")\n",
+    "\n",
+    "g3.plot()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ea1c3015",
+   "metadata": {},
+   "source": [
+    "Data citation:\n",
```

Comment:
Last cell should be Call to Action, imagine someone jumped here from a blogpost and wants to learn more --

* Kusto Graph
* 10 Min to Graphistry (viz)
* 10 Min to GFQL (graph query)
* UMAP?
* Louie.ai ?